composer.lock

```diff
@@ -497,7 +520,69 @@
                     "type": "community_bridge"
                 }
             ],
-            "time": "2021-01-24T20:45:49+00:00"
+            "time": "2021-07-14T13:59:23+00:00"
+        },
+        {
+            "name": "laminas/laminas-zendframework-bridge",
```

Comment:
`laminas/laminas-zendframework-bridge` still made it in here: let's try getting rid of it before merge

Reply:
We can't - it's from laminas-mail. So until both have their release for 8.1, this one will have the bridge. It'll have to wait until the next minor after this one, or next patch release, if we update the lockfile then.

Reply:
got it, makes sense